src/super_gradients/training/sg_trainer/sg_trainer.py

```diff
@@ -2092,6 +2097,7 @@ def test(
         metrics_progress_verbose=False,
         test_phase_callbacks=None,
         use_ema_net=True,
+        max_batches=None,
```

Comment:
Even though name is pretty self explanatory, please add docs for max_batches.

Reply:
TY, Added